### ISSUE TITLE
No more CSS in commits

### ISSUE THREAD

**Issue by huntons** [wunderkraut/WunderWay#96]
SASS only!

**Comment by henrijs** [comment]
For this we need to add some text for developers about how to get css for local development. One page for that hopefully be enough.

**Comment by lewisnyman** [comment]
+1 to this. I've always found this to be more of a devops problem? Do we need a common setup that compiles Sass on the server?

**Comment by joesb** [comment]
Yes, I think that's the thought behind this issue. I.e. if we do remove CSS files from themes, then we need to let the 'machines' that display site pages need to be able to (re)compile on command - on the dev/stage/production servers, and also on developers' local machines - and make sure (backend) devs are not frightened of that, know how to do it quickly, etc.

**Comment by henrijs** [comment]
Compile on build has been working on platform.sh. Very slick. Challenge is local environments and non-them-er developers. Some of them hate theme stack with passion :)
